cocos/core/platform/debug.ts

```diff
@@ -59,7 +61,7 @@ function formatString (message?: any, ...optionalParams: any[]) {
  * @param optionalParams - JavaScript objects with which to replace substitution strings within msg.
  * This gives you additional control over the format of the output.
  */
-export function log (message?: any, ...optionalParams: any[]) {
```

Comment:
Should not change these coding style

Reply:
It is set in eslint config file
![image](https://user-images.githubusercontent.com/40414978/208012239-e39407aa-f23b-4077-a381-f4e3f80c5c81.png)

https://eslint.org/docs/latest/rules/space-before-function-parentheses

Reply:
I missed

Reply:
fixed